lucene/analysis/common/src/test/org/apache/lucene/analysis/hunspell/i53643.good

```diff
@@ -0,0 +1,21 @@
+1
+12
+123
+1234
+12345
+123456
+1234567
+1.1
+1.12
+1.123
+1.1234
+1.12345
+1.123456
+12.1
+123.12
+1234.123
+12345.1234
+123456.12345
+1234567.123456
+4,2
```

Comment:
Yeah.... this is really locale-specific. Can't be solved accurately without the context (locale). Even if you do have the locale, people write inconsistently (grouping and fraction separators).

Reply:
Very specific, yes. That's how Hunspell behaves: https://github.com/hunspell/hunspell/blob/master/src/hunspell/hunspell.cxx#L501